website/source/docs/auth/aws-ec2.html.md

```diff
@@ -843,9 +848,10 @@ in its identity document to match the one specified by this parameter.
       <li>
         <span class="param">bound_iam_role_arn</span>
         <span class="param-flags">optional</span>
-If set, defines a constraint on the EC2 instances to be associated with an IAM
-role ARN which has a prefix that matches the value specified by this
-parameter. Note that an exact match is also a prefix.
+	If set, defines a constraint on the authenticating EC2 instance that it
+must IAM role ARN specified by this parameter. Note that an exact match is also
```

Comment:
We generally attempt to keep the descriptions the same at both the website the the actual path descriptions on the code. There are religious `vault path-help` users who refer to that just as much as the website :-)